core/src/main/java/io/grpc/internal/ServerImpl.java

```diff
@@ -508,21 +512,41 @@ public void cancelled(Context context) {
     private <ReqT, RespT> ServerStreamListener startCall(ServerStream stream, String fullMethodName,
         ServerMethodDefinition<ReqT, RespT> methodDef, Metadata headers,
         Context.CancellableContext context, StatsTraceContext statsTraceCtx) {
-      // TODO(ejona86): should we update fullMethodName to have the canonical path of the method?
```

Comment:
Why was this comment deleted?

Reply:
This got mangled when fixing the rebase conflicts, fixed.